.github/workflows/pr-integration.yml

```diff
@@ -10,15 +10,25 @@ on:
     branches:
       - "main"
       - "release-*"
-    paths-ignore:
-      - 'docs/**'
-      - 'dependencies/**'
-      - '**/*.md'
-      - 'OWNERS'
-      - 'CODEOWNERS'
-      - 'external-images.yaml'
-  workflow_dispatch:
+    # only include PRs that change:
+    # go files
+    # go mod files
+    # test files in the test/e2e directory
+    # test files in the istio directory
+    # exclude go files that end with _test.go
+    # include the workflow definition itself
+    # include dependencies of the workflow definition
+    paths:
```

Comment:
the kustomize files are indirectly tested as well in the e2e tests and might be included, what do you think?
besides this I guess we are good to go

Reply:
I included config/**
there are still options to exclude certain files (busola etc.) from the files that trigger tests...